sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_mldesigner/__init__.py

```diff
@@ -0,0 +1,121 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""
+This file stores functions and objects that will be used in mldesigner package.
+DO NOT change the module names in "all" list. If the interface has changed in source code, wrap it here and keep
+original function/module names the same as before, otherwise mldesigner will be broken by this change.
+"""
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
+
+from azure.ai.ml import(
+    Input,
+    Output,
+    MLClient,
+    load_component,
+    MpiDistribution,
+    PyTorchDistribution,
+    TensorFlowDistribution,
+)
+from azure.ai.ml.entities import(
+    Component,
+    CommandComponent,
+    PipelineComponent,
+    Environment,
+    ResourceConfiguration,
+    ValidationResult,
+    Command,
+    Pipeline,
+    Parallel,
+)
+from azure.identity import AzureCliCredential
+from azure.ai.ml.exceptions import ErrorTarget, UserErrorException
+from azure.ai.ml.entities._component.component_factory import component_factory
+from azure.ai.ml.entities._job.pipeline._load_component import _generate_component_function
+from azure.ai.ml._internal.entities._additional_includes import _AdditionalIncludes
+from azure.ai.ml._utils.utils import load_yaml
+from azure.ai.ml.entities._validation import SchemaValidatableMixin
+from azure.ai.ml._schema import PathAwareSchema
+from azure.ai.ml.dsl._utils import _sanitize_python_variable_name
+from azure.ai.ml._internal.entities import (
+    InternalComponent,
```

Comment:
Let's open a corresponding mldesigner PR, so we can decide whether we can avoid some these imports

Reply:
Sure